src/lib/network/clients/network_proposal.rb

```diff
@@ -74,11 +75,19 @@ def ask_user(args)
 
   private
 
+    def config
+      Yast::Lan.yast_config
```

Comment:
Maybe you can use `Yast::Lan.yast_config` directly in the proposal summary.

Reply:
Well, it currently uses the same config that is modified during the installation but could be the autoYaST one or maybe one from the proposal and other for the running system so I prefer to pass it as a parameter to the presenter as we do with others.